sci-libs/cantera/files/cantera-2.5.1_env.patch

```diff
@@ -45,6 +45,50 @@ diff -Naur old/interfaces/cython/SConscript new/interfaces/cython/SConscript
      else:
          extra = '--user'
          localenv.AppendENVPath(
+diff -Naur old/interfaces/python_minimal/SConscript new/interfaces/python_minimal/SConscript
```

Comment:
Try[ clean up](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto) patches if you can by removing the diff lines, using a/ and b/, and removing dates.

Reply:
(I'll fix this myself now to get it merged.)

Reply:
I sent non-Gentoo specific parts of patch to upstream too.

Reply:
Excellent as ever. :)